skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -92,10 +123,32 @@ def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
     
     See Also
     --------
-    skimage.measure.marching_cubes
+    skimage.measure.marching_cubes_classic
     skimage.measure.mesh_surface_area
     """ 
     
+    # This signature (output args) of this func changed after 0.12
+    try:
+        nout = expected_output_args()
+    except Exception:
```

Comment:
This scares me: what kind of exceptions are we expecting here? Why can't this be more specific?